src/Package/Impl/ExportDialog/ExportImageDialog.xaml

```diff
@@ -0,0 +1,52 @@
+﻿<ui:PlatformDialogWindow x:Class="Microsoft.VisualStudio.R.Package.ExportDialog.ExportImageDialog"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:pkg="clr-namespace:Microsoft.VisualStudio.R.Package"
+        xmlns:ui="clr-namespace:Microsoft.VisualStudio.R.Package.Wpf"  
+        xmlns:wpf="clr-namespace:Microsoft.R.Wpf;assembly=Microsoft.R.Wpf"
+        xmlns:local="clr-namespace:Microsoft.VisualStudio.R.Package.ExportDialog"
+        xmlns:components="clr-namespace:Microsoft.R.Components;assembly=Microsoft.R.Components"
+        mc:Ignorable="d" MinHeight="150" x:Name="Self"
+        ShowInTaskbar="False" WindowStyle="SingleBorderWindow"
+        Background="{DynamicResource {x:Static wpf:Brushes.BackgroundBrushKey}}"
+        Foreground="{DynamicResource {x:Static wpf:Brushes.UITextKey}}"
+        Title="{x:Static pkg:Resources.ExportImageDialog_Title}" SizeToContent="WidthAndHeight">
+    <Grid Margin="0,0,0,-3">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto"/>
+            <RowDefinition />
+            <RowDefinition Height="auto"/>
+            <RowDefinition Height="auto"/>
+        </Grid.RowDefinitions>
+        <Grid Grid.Row="0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition/>
+                <ColumnDefinition/>
+                <ColumnDefinition Width="auto"/>
+            </Grid.ColumnDefinitions>
+            <StackPanel  Orientation="Horizontal" HorizontalAlignment="Left">
+                <TextBlock  FontWeight="Bold" Text="{x:Static pkg:Resources.label_Width}" VerticalAlignment="Center" Margin="10,10"/>
```

Comment:
Too many panels for a simple dialog box. One top level grid with required amount of rows and columns is enough.